src/servers/sapphire_zone/DebugCommand/DebugCommandHandler.cpp

```diff
@@ -768,6 +768,45 @@ void Core::DebugCommandHandler::instance( char* data, Entity::Player &player, bo
 
       instance->setVar( static_cast< uint8_t >( index ), static_cast< uint8_t >( value ) );
    }
+   else if( subCommand == "objupdate" )
+   {
+      uint32_t instanceId;
+      uint32_t objId;
+
+      sscanf( params.c_str(), "%d %d", &instanceId, &objId );
+
+      auto pInstance = g_territoryMgr.getInstanceZonePtr( instanceId );
+      if( !pInstance )
```

Comment:
I feel like adding error messages to each of these silent returns would be nice, to see exactly what went wrong (those debug commands could be used just for checking if something's working as intended for example). what do you think?

Reply:
yeah I thought about this but I don't think its worthwhile, mainly because these are likely to change soon anyway given that the current stuff is very wip